frontend/src/players_view/PlayersView.jsx

```diff
@@ -12,9 +12,38 @@ function PlayersView() {
   const [playersDisplayed, setPlayersDisplayed] = useState([]);
   const [page, setPage] = useState(1);
   const [rowsPerPage, setRowsPerPage] = useState(10);
-  const [sortType, setSortType] = useState("");
+  const [sortType, setSortType] = useState("no_sort");
+  const [sortDirection, setSortDirection] = useState("no_direction");
```

Comment:
Attempted to emulate Typescript enums with JS Object.freeze. Problem: because we're setting sortType through the id of the option's component, we're passing a String into setSortType, so it's not actually doing anything different. Internet says that a workaround is using String template literal when declaring usestate (eg = useState<`SORT_TYPE`>(SORT_TYPE.no_sort). I don't know much about templates but it seems like that's also specific to Typescript. You can check "attempt js enums" commit in today's pr for details on last attempt; will revisit.